src/components/BottomNavigationBot/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, memo } from 'react'
+import ModalContainer from '../ModalContainer'
+import './styles.scss'
+import botIcon from '../../assets/bot.svg'
+
+const BottomNavigationBot = () => {
+  const [showBot, setShowBot] = useState(false)
+  const openModal = () => setShowBot(true)
+  const closeModal = () => setShowBot(false)
+
+  return (
+    <div className="nav-bot">
+      <button type="button" className="button-image" onClick={openModal}>
+        <img src={botIcon} alt="Botti" />
+      </button>
+      <ModalContainer
+        modalIsOpen={showBot}
+        closeModal={closeModal}
+        label="Botti"
+      >
+        <div className="modal-content">
+          <div className="modal-item">
+            <i className="fas fa-question-circle modal-icon" />
```

Comment:
Also icons should probably have aria-hidden set true for screen readers since they are only decorative here. Check out [Using the aria-hidden attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute)